oat-sa/extension-lti-outcomeui
# feature/MS-988/ audio response in lti outcome UI

Ticket: https://oat-sa.atlassian.net/browse/MS-988

- Added the file response in ltiOutcomeUi

### How to test?

- Create an audio PCI item
- Create a test, publish it  and take the test
- Create an LTI provider
- Go manage roles from the top-right menu > LTI Roles > Instructor > select Test Taker role and save
- Go to http://ltiapps.net/test/tc.php and add the launch URL(`https://YOUR_DOMAIN/ltiOutcomeUi/ItemResultPreviewer/launch?resultId=YOUR_RESULT_URI&itemRef=YOU_ITEM_REF`), key and secret of the LTI provider, add the `instructor` role and launch it
- Audio PCI should work as expected in review mode with LTI
